logical/framework/field_type.go

```diff
@@ -13,6 +13,16 @@ const (
 	// TypeDurationSecond represent as seconds, this can be either an
 	// integer or go duration format string (e.g. 24h)
 	TypeDurationSecond
+
+	// TypeSlice represents a slice of any type
+	TypeSlice
+	// TypeStringSlice is a helper for TypeSlice that returns a sanitized
```

Comment:
This comment seems a little confusing -- what does "sanitized" mean? Probably you meant type-cast but it evokes questions about lowercase/uppercasing, de-duplication, etc. since many fields in Vault go through that.